### PR TITLE
2nd milestone

### PR DESCRIPTION
**Pull request by ricardo123321**
added initial components and displayed the numbers on the screen

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hi @ricardo123321  👋 

**You've done quite well here. 👌**

- Please refer to comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+  render() {
+    return (
+      <div className="App">
+        <Display />
+        <ButtonPanel />
+      </div>
+    );
+  }
+}
```
Comment (mikenath223) [inline src/components/App.js]: - [ ] Kindly change this to a functional component.

File: src/components/App.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="App">
+        <Display />
+        <ButtonPanel />
+      </div>
```
Comment (mikenath223) [inline src/components/App.js]: - [ ] Also please wrap Display and ButtonPanel components in a `React.Fragment` shorter syntax.

File: src/components/ButtonPanel.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    <div>
+      <Button name="1" />
+      <Button name="2" />
+      <Button name="3" />
+      <Button name="+" />
+    </div>
+    <div>
+      <Button name="0" />
+      <Button name="." />
+      <Button name="=" />
```
Comment (mikenath223) [inline src/components/ButtonPanel.js:33]: - [ ] OPTIONAL - Perhaps consider grouping these buttons into an array of objects then mapping through them. It's terser that way.

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @ricardo123321 :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**